conf/resources/location-autocomplete-canonical-list.json

```diff
@@ -0,0 +1,1114 @@
+[
```

Comment:
Is this json file needed?

Reply:
I'm not convinced after comparing the produced country list with that used by ALF and them being the same. I'm going to ditch it.